p2p/pex_reactor.go

```diff
@@ -293,6 +309,160 @@ func (r *PEXReactor) ensurePeers() {
 	}
 }
 
+// Explores the network searching for more peers. (continuous)
+// Seed/Crawler Mode causes this node to quickly disconnect
+// from peers, except other seed nodes.
+func (r *PEXReactor) seedCrawlerMode() {
+	// Do an initial crawl
+	r.crawlPeers()
+
+	// Fire periodically
```

Comment:
code is self describing enough. this comment only duplicates code

Reply:
The same type of comment can be found in ensurePeersRoutine(), which is where the idea came from, but I'll remove it.